test/Psr7ResponseTest.php

```diff
@@ -45,7 +49,31 @@ public function testResponseToZend($body, $status, $headers)
 
         $zendResponse = Psr7Response::toZend($psr7Response);
         $this->assertInstanceOf(ZendResponse::class, $zendResponse);
-        $this->assertEquals($body, (string) $zendResponse->getBody());
+        $this->assertEquals($body, (string)$zendResponse->getBody());
+        $this->assertEquals($status, $zendResponse->getStatusCode());
+
+        $zendHeaders = $zendResponse->getHeaders()->toArray();
+        foreach ($headers as $type => $values) {
+            foreach ($values as $value) {
+                $this->assertContains($value, $zendHeaders[$type]);
+            }
+        }
+    }
+
+    /**
+     * @dataProvider getResponseData
+     */
+    public function testResponseToZendFromRealStream($body, $status, $headers)
+    {
+        $stream = new Stream(__DIR__ . '/../composer.json', 'wb+');
```

Comment:
Please don't use the `composer.json` file, use instead a random file generated in a temporary dir using [sys_get_temp_dir()](http://php.net/manual/en/function.sys-get-temp-dir.php) or a fixed file stored in a `test/TestAsset/` folder.

Reply:
good catch! updated to use `tempnam(sys_get_temp_dir(), 'Test')`